modules/xsession.nix

```diff
@@ -57,6 +57,12 @@ in
         description = "Extra shell commands to run before session start.";
       };
 
+      lockCmd = mkOption {
+        type = types.str;
+        description = "Command to use to lock the screen";
+        default = "${pkgs.xscreensaver}/bin/xscreensaver";
```

Comment:
same remark rycee made in other PR, add defaultStr to have the doc independant of revision.

Reply:
Fixed